aiohttp_admin/templates/admin_on_rest.jinja2

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    {% block head %}
+    <meta charset="UTF-8">
+    <title>{{ name }} Admin</title>
+    {% endblock %}
+    <!-- main application data -->
+    <script type="text/javascript">
+        window.appData = {
+          tokenUrl: "{{ url('admin.token') }}",
+          tokenName: "aiohttp_admin_token",
+          loginUrl: "{{ url('admin.token') }}",
```

Comment:
what the difference between loginUrl and tokenUrl ?

Reply:
I could not choose which is better) thx i will remove some